### ISSUE TITLE
GraphQL: Image or File field types missing GraphQL fields

### ISSUE THREAD

**Issue by dsilvam** [dotCMS/core#18005]
Related to https://github.com/dotCMS/core/issues/17560. 

If you have a Content Type with a field of type `Image` or `File` then GraphQL fields that were available before are not anymore. 

They are needed to be available again.

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/core/pull/18009

**Comment by dsilvam** [comment]
Note to QA: Add  `File` and `Image` fields to an existing or new content type and make sure you have the following GraphQL fields available for them:
* fileName
* description
* fileAsset (Composed/Custom Type. see `Binary` type on our GraphQL doc)
* metaData (Custom Type. See `Key Value` type on our GraphQL doc)
* showOnMenu
* sortOrder

Example query:
```
{
  blogCollection {
    myImage {
      fileAsset {
        name
        size
        mime
      }
      metaData {
        key
        value
      }
      description
    }
    
    myFile {
      fileAsset {
        name
        size
        mime
      }
      metaData {
        key
        value
      }
      description
    }

  }
}

```

**Comment by dsilvam** [comment]
PR 2: https://github.com/dotCMS/core/pull/18030

**Comment by nollymar** [comment]
**Tests Results:** Success. It was tested using new and existing content types. All these fields were validated: 

- fileName
- description
- fileAsset
- metaData
- showOnMenu
- sortOrder

**Comment by bryanboza** [comment]
Fixed, tested on release-5.2.7 // Postgres // FF and now we are able to get this info from those fields
